.github/workflows/test.yml

```diff
@@ -8,31 +8,31 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Install Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
```

Comment:
Our [GHA recommendations guide](https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas) has a section on pinning actions to SHAs, for security purposes. Admittedly, the `actions/` actions are less likely to be affected, but this workflow also uses some workflows developed by third-parties, for which I'd definitely recommend pinning them. It's totally fine to do this in a follow-up PR.